src/index.js

```diff
@@ -1,7 +1,18 @@
 import './style.css';
+import REQUESTS from '../modules/api.js';
 
-import UI from '../modules/populate.js';
+const form = document.querySelector('#form');
+const refreshBtn = document.querySelector('.refresh');
 
 document.addEventListener('DOMContentLoaded', () => {
-  UI.display();
+  REQUESTS.displayScores();
+});
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  REQUESTS.createNewScore();
+  REQUESTS.clearForm();
+});
+refreshBtn.addEventListener('click', () => {
+  REQUESTS.displayScores();
```

Comment:
I love the way you arrange your codes rightly using the single responsibility principle. 👏🏾 

- [ ] [OPTIONAL] Please consider always specifying either production or development for the `mode` option in your webpack.config.js file to prevent this error warning message.

Error message
--- | 
![image](https://user-images.githubusercontent.com/84156714/223724513-ff203340-e3ab-49cc-909e-3de1930fced6.png)

I have to add the mode option and set it to development as shown below 👇🏾 

```javascript
module.exports = {
  entry: {
    index: './src/index.js',
  },
  mode: "development", // I added this. 
  devServer: {
    static: './dist',
  },
  // ... other options

};

```

Reply:
Thank you so much advice @bobb-Rob